server.py

```diff
@@ -34,13 +34,20 @@
 from utils import url_fails
 from utils import get_video_duration
 
-from settings import settings, DEFAULTS
 from werkzeug.wrappers import Request
+
+from settings import DEFAULTS
+from settings import config_dir
+from settings import config_file
+from settings import load as load_settings
+from settings import save as save_settings
+
```

Comment:
What "load_setting" actually does? I don't think that performing actions silently during module load is good idea.